interface/main/calendar/modules/PostCalendar/pnadmin.php

```diff
@@ -441,18 +440,18 @@ function postcalendar_admin_categories($msg = '', $e = '', $args = array())
         $template_name ='default';
     }
 
-    $output->Text(postcalendar_adminmenu());
+    $output->Text(postcalendar_adminmenu("category"));
 
     if (!empty($e)) {
-        $output->Text('<div style="padding:5px; border:1px solid red; background-color: pink;">');
-        $output->Text('<center><b>' . text($e) . '</b></center>');
-        $output->Text('</div><br />');
+        $output -> Text('<div class="alert alert-danger ml-5 mr-5" role="alert">');
```

Comment:
This should be `ml-1 mr-1`. Padding isn't measured in pixels in Bootstrap. [Please read the documentation on how to do this correctly](https://getbootstrap.com/docs/4.4/utilities/spacing/).